precompile/contracts/rewardmanager/module.go

```diff
@@ -0,0 +1,63 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package rewardmanager
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/subnet-evm/precompile/config"
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ava-labs/subnet-evm/precompile/modules"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// ConfigKey is the key used in json config files to specify this precompile config.
+// must be unique across all precompiles.
+const ConfigKey = "rewardManagerConfig"
+
+var ContractAddress = common.HexToAddress("0x0200000000000000000000000000000000000004")
+
+var Module = modules.Module{
+	ConfigKey:    ConfigKey,
+	Address:      ContractAddress,
+	Contract:     RewardManagerPrecompile,
+	Configurator: &configuror{},
+}
+
+type configuror struct{}
+
+func init() {
+	if err := modules.RegisterModule(Module); err != nil {
+		panic(err)
+	}
+}
+
+func (*configuror) NewConfig() config.Config {
+	return &Config{}
+}
+
+// Configure configures [state] with the desired admins based on [cfg].
+func (*configuror) Configure(chainConfig contract.ChainConfig, cfg config.Config, state contract.StateDB, _ contract.BlockContext) error {
+	config, ok := cfg.(*Config)
+	if !ok {
+		return fmt.Errorf("incorrect config %T: %v", config, config)
+	}
+	// TODO: should we move this to the end and return it for consistency with other precompiles?
```

Comment:
Do you just mean consistency in that `AllowListConfig.Configure(...)` is usually called at the end as opposed to the beginning?

We can, I don't think it's required to do so.

Reply:
moved